openedx/openedx-platform
# fix: remove error ReferenceError: JavascriptLoader is not defined in Zoom Image Tool

## Description

This PR uses a workaround to load the script [jquery.loupeAndLightbox.js](https://files.edx.org/jquery.loupeAndLightbox.js), which waits for the script to be completely loaded in the browser before running any reference to its utilities.

Useful information to include:

- Must be update [RTD](https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/exercises_tools/zooming_image.html)
- Check CSS Style `.zooming-image-place .larger` in [frontend-app-learning](https://github.com/openedx/frontend-app-learning)

**LMS In Firefox:**
![Captura de pantalla de 2024-10-16 23-10-56](https://github.com/user-attachments/assets/9507725d-02ca-437e-95ba-924e232e62ef)

**CMS in Firefox:**
![Captura de pantalla de 2024-10-16 23-27-39](https://github.com/user-attachments/assets/433ff80c-2822-416b-95d2-fc2a32a37d9d)

**LMS in Chrome:**
![Captura de pantalla de 2024-10-16 23-09-26](https://github.com/user-attachments/assets/f10f0e52-65b0-4be4-be6a-90d9e2e661b4)

**CMS in Chrome:**
![Captura de pantalla de 2024-10-16 23-09-21](https://github.com/user-attachments/assets/72570220-51f7-4a70-b0d4-502514a6e664)

## Supporting information

[DEPR] HTML Block: The Zooming Image Tool template is broken: https://github.com/openedx/edx-platform/issues/31436

## Testing instructions

1. Fire up a devstack, open the demo course in Studio and add a Text block to any unit.
2. Select the Zooming Image Tool template and publish the unit; there's no need to edit the contents.
3. Test the resulting tool in Studio itself: the zoom should work as expected.
4. Open the unit in the LMS. Clicking on the image will open the utility that allows you to view the magnified image in a loupe.

## Deadline

None

## Other information

None
